jbang-catalog.json

```diff
@@ -30,6 +30,15 @@
     },
     "kill": {
       "script-ref": "kill.java"
+    },
+    "quarkus-format": {
```

Comment:
shouldn't we stick to `fmt` or `format` ?

My vote go for: `quarkus-fmt` and `fmt`

Reply:
Let's stick with `quarkus-fmt` and `fmt` for now